<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QTabWidget" name="repoTab">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="officialTab">
      <attribute name="icon">
       <iconset>
        <normaloff>:/Icons/icons/archlinux-icon-crystal-32.svg</normaloff>:/Icons/icons/archlinux-icon-crystal-32.svg</iconset>
      </attribute>
      <attribute name="title">
       <string>Arch Linux Official Repositories</string>
      </attribute>
      <layout class="QGridLayout" name="_15">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="_16">
         <item>
          <layout class="QVBoxLayout" name="_17">
           <item>
            <layout class="QHBoxLayout" name="_18">
             <item>
              <widget class="QLabel" name="label_18">
               <property name="maximumSize">
                <size>
                 <width>150</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap>:/Icons/icons/archlinux-icon-crystal-32.svg</pixmap>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="officialLabel">
               <property name="text">
                <string>Here you can choose what repositories to use between Arch Linux's official ones. Please note that packages from Unstable and Testing may break your system.</string>
               </property>
               <property name="scaledContents">
                <bool>true</bool>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QCheckBox" name="coreBox">
             <property name="text">
              <string comment="Do not translate">Core</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="extraBox">
             <property name="text">
              <string comment="Do not translate">Extra</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="communityBox">
             <property name="text">
              <string comment="Do not translate">Community</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="testingBox">
             <property name="text">
              <string comment="Do not translate">Testing</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>Server</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLineEdit" name="addMirrorLine"/>
              </item>
              <item>
               <widget class="QPushButton" name="addMirrorButton">
                <property name="text">
                 <string>Add Mirror</string>
                </property>
                <property name="icon">
                 <iconset>
                  <normaloff>:/Icons/icons/list-add.png</normaloff>:/Icons/icons/list-add.png</iconset>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QScrollArea" name="archScrollArea">
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="scrollAreaWidgetContents_2">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>572</width>
                 <height>289</height>
                </rect>
               </property>
              </widget>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="addServerButton">
                <property name="text">
                 <string>Add another server</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="KDEModTab">
      <attribute name="icon">
       <iconset>
        <normaloff>:/Icons/icons/archlinux-mods-kdemod.svg</normaloff>:/Icons/icons/archlinux-mods-kdemod.svg</iconset>
      </attribute>
      <attribute name="title">
       <string>KDEMod Repositories</string>
      </attribute>
      <layout class="QVBoxLayout" name="_19">
       <item>
        <layout class="QVBoxLayout" name="_20">
         <item>
          <layout class="QHBoxLayout" name="_21">
           <item>
            <widget class="QLabel" name="label_15">
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap>:/Icons/icons/archlinux-mods-kdemod.svg</pixmap>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="KDEModLabel">
             <property name="text">
              <string>KDEMod is an improved version of KDE, an Arch Linux exclusive. Featuring an improved Qt build, compatible with Arch Linux standard Qt's, an improved theme, and lots of addons, KDEMod is the best way to taste the ease and the power of KDE with the simplicity and the power of Arch.</string>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="KDEMod4Box">
           <property name="text">
            <string>KDEMod 4 Repository</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="KDEMod4ExtragearBox">
           <property name="text">
            <string>KDEMod 4 Extragear Repository</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="KDEMod4PlaygroundBox">
           <property name="text">
            <string>KDEMod 4 Playground Repository</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="KDEMod4TestingBox">
           <property name="text">
            <string>KDEMod 4 Testing Repository</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="KDEMod4UnstableBox">
           <property name="text">
            <string>KDEMod 4 Unstable Repository (Testers only!)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="KDEMod3Box">
           <property name="text">
            <string>KDEMod 3 Repository</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_13">
           <property name="title">
            <string>Server</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLineEdit" name="addMirrorLine_2"/>
              </item>
              <item>
               <widget class="QPushButton" name="addMirrorButton_2">
                <property name="text">
                 <string>Add Mirror</string>
                </property>
                <property name="icon">
                 <iconset>
                  <normaloff>:/Icons/icons/list-add.png</normaloff>:/Icons/icons/list-add.png</iconset>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QScrollArea" name="kdemodScrollArea">
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="scrollAreaWidgetContents_3">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>350</width>
                 <height>63</height>
                </rect>
               </property>
              </widget>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="addKDEModServerButton">
                <property name="text">
                 <string>Add another server</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="thirdPartyTab">
      <attribute name="icon">
       <iconset>
        <normaloff>:/Icons/icons/network-server-database.png</normaloff>:/Icons/icons/network-server-database.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Third-Party Repositories</string>
      </attribute>
      <layout class="QGridLayout" name="_22">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="_23">
         <item>
          <widget class="QLabel" name="thirdPartyLabel">
           <property name="text">
            <string>You can extend your packages choice by adding other repositories. Third-Party Repositories act just as normal ones, and will be accessible through both Shaman and Pacman.</string>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QScrollArea" name="scrollArea">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>368</width>
              <height>335</height>
             </rect>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="_24">
           <item>
            <spacer>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>71</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="addThirdPartyButton">
             <property name="text">
              <string>Add New Repository</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>:/Icons/icons/list-add.png</normaloff>:/Icons/icons/list-add.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Repository Order</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>20</number>
         </property>
         <property name="topMargin">
          <number>15</number>
         </property>
         <item>
          <widget class="QLabel" name="label_23">
           <property name="text">
            <string>Please order your repositories to define which one will be preferred over another.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listWidget_2">
           <property name="dragDropMode">
            <enum>QAbstractItemView::InternalMove</enum>
           </property>
           <property name="movement">
            <enum>QListView::Free</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
